supplementary/devices/taoshan.rst

```diff
@@ -0,0 +1,75 @@
+
+Taoshan - Sony Xperia L
+===========================
+
+**//Disclaimer:** Please remove everything that is just part of this template and not actually about the device before submitting this into the official wiki, thank you! **//**
```

Comment:
The disclaimer clearly says to remove it before submitting to the official wiki. Please, do it

Reply:
Sorry. Should not have opened PR back then